openff/recharge/esp/_esp.py

```diff
@@ -179,6 +183,9 @@ def generate(
             Whether to compute the ESP at each grid point.
         compute_field
             Whether to compute the field at each grid point.
+        memory
+            The memory to make available to Psi4 for computation
```

Comment:
Teensiest nit (which can be passed on): future me would benefit from dropping in this link to know why this default value was used

https://psicode.org/psi4manual/master/psithoninput.html#memory-specification

Reply:
Thanks for the review Matt -- added!